perfmetrics/scripts/testing_on_gke/examples/fio/loading-test/templates/fio-tester.yaml

```diff
@@ -58,7 +63,7 @@ spec:
         echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
         apt-get update && apt-get install -y google-cloud-cli
 
-        gsutil -m cp -R gs://{{ .Values.bucketName }}/* /data 
+        gsutil -m cp -R gs://{{ .Values.bucketName }}/* /data
```

Comment:
Since gsutil will soon be deprecated, we should use gcloud instead.

Reply:
done, replaced all instances of gsutil with gcloud.